modules/table/src/test/java/org/apache/ignite/internal/table/distributed/index/IndexBuilderTest.java

```diff
@@ -50,6 +50,8 @@ public class IndexBuilderTest extends BaseIgniteAbstractTest {
 
     private static final int PARTITION_ID = 3;
 
+    private static final long ANY_ENLISTMENT_CONSISTENCY_TOKEN = 100500;
```

Comment:
We've also agreed that you will adjust PrimaryReplicaMissException. I cant see such changes in given PR.

Reply:
Do as we personally discussed, but unfortunately I still can’t get rid of the `org.apache.ignite.internal.replicator.exception.PrimaryReplicaMissException#PrimaryReplicaMissException(java.lang.String, java.lang.String)` because it is used in `org.apache.ignite.internal.table.distributed.replicator.PartitionReplicaListener#processTxStateCommitPartitionRequest` and I don’t know how to fix it correctly.

Reply:
We discussed it personally: there will be only one constructor, I will adapt to it.